{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.1",
	"cveMetadata": {
		"cveId": "CVE-2021-20219",
		"assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
		"assignerShortName": "redhat",
		"state": "PUBLISHED",
		"datePublished": "2021-03-23T16:34:00",
		"dateUpdated": "2024-08-03T17:30:07.576Z",
		"dateReserved": "2020-12-17T00:00:00"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "53f830b8-0a3f-465b-8143-3b8a9948e749",
				"shortName": "redhat",
				"dateUpdated": "2021-03-23T16:34:00"
			},
			"descriptions": [
				{
					"lang": "en",
					"value": "A denial of service vulnerability was found in n_tty_receive_char_special in drivers/tty/n_tty.c of the Linux kernel. In this flaw a local attacker with a normal user privilege could delay the loop (due to a changing ldata->read_head, and a missing sanity check) and cause a threat to the system availability."
				}
			],
			"affected": [
				{
					"vendor": "n/a",
					"product": "kernel",
					"versions": [
						{
							"status": "affected",
							"version": "kernel 4.18"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "CWE",
							"lang": "en",
							"description": "CWE-697",
							"cweId": "CWE-697"
						}
					]
				}
			],
			"references": [
				{
					"tags": [
						"x_refsource_MISC"
					],
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1923738"
				}
			],
			"x_legacyV4Record": {
				"CVE_data_meta": {
					"ASSIGNER": "secalert@redhat.com",
					"ID": "CVE-2021-20219",
					"STATE": "PUBLIC"
				},
				"affects": {
					"vendor": {
						"vendor_data": [
							{
								"product": {
									"product_data": [
										{
											"product_name": "kernel",
											"version": {
												"version_data": [
													{
														"version_value": "kernel 4.18"
													}
												]
											}
										}
									]
								},
								"vendor_name": "n/a"
							}
						]
					}
				},
				"data_format": "MITRE",
				"data_type": "CVE",
				"data_version": "4.0",
				"description": {
					"description_data": [
						{
							"lang": "eng",
							"value": "A denial of service vulnerability was found in n_tty_receive_char_special in drivers/tty/n_tty.c of the Linux kernel. In this flaw a local attacker with a normal user privilege could delay the loop (due to a changing ldata->read_head, and a missing sanity check) and cause a threat to the system availability."
						}
					]
				},
				"problemtype": {
					"problemtype_data": [
						{
							"description": [
								{
									"lang": "eng",
									"value": "CWE-697"
								}
							]
						}
					]
				},
				"references": {
					"reference_data": [
						{
							"name": "https://bugzilla.redhat.com/show_bug.cgi?id=1923738",
							"refsource": "MISC",
							"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1923738"
						}
					]
				}
			}
		},
		"adp": [
			{
				"providerMetadata": {
					"orgID": "af854a3a-2127-422b-91ae-364da2661108",
					"shortName": "CVE",
					"dateUpdated": "2024-08-03T17:30:07.576Z"
				},
				"title": "CVE Program Container",
				"references": [
					{
						"tags": [
							"x_refsource_MISC",
							"x_transferred"
						],
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1923738"
					}
				]
			}
		]
	}
}
